awx/main/utils/common.py

```diff
@@ -842,14 +843,13 @@ def get_corrected_memory(memory):
     return memory
 
 
-def get_mem_effective_capacity(mem_bytes):
+def get_mem_effective_capacity(mem_bytes, node_type=None, is_execution_node=False):
```

Comment:
same as get_cpu_effective_capacity, we should change to `is_control_node` signature